src/compile.c

```diff
@@ -850,10 +850,16 @@ dcc_build_somewhere(char *argv[],
                                   needs_dotd ? deps_fname : NULL,
                                   server_stderr_fname,
                                   cpp_pid, local_cpu_lock_fd,
-                  host, status)) != 0) {
+                  host, status, &remote_unsupported)) != 0) {
```

Comment:
I don't totally love adding another state variable and goto but I guess it's ok.

Reply:
I can assign some magic value to `status` and check for that instead. Would this make the code more clear?

Reply:
I think it's OK like this, thanks.